cmd/orgs.go

```diff
@@ -44,6 +51,17 @@ func init() {
 					setUserEnv, checkRequiredFlags, orgsRemove,
 				),
 			},
+			{
+				Name:  "members",
+				Usage: "List all members in an org",
+				Flags: []cli.Flag{
+					orgFlag("Use this organization.", false),
+				},
+				Action: chain(
+					ensureDaemon, ensureSession, loadDirPrefs, loadPrefDefaults,
+					setUserEnv, checkRequiredFlags, orgsMembersListCmd,
```

Comment:
I don't think we need to `setUserEnv` or `checkRequiredFlags` here as the `orgFlag` is not required and we don't use the environment flag at all.